resources/definitions/maker_made_300x.def.json

```diff
@@ -0,0 +1,152 @@
+{
+    "version": 2,
+    "name": "Maker Made 300x",    
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "DragonJe",
+        "manufacturer": "Maker Made",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [0, 0, 0],
+        "has_materials": true,
+        "has_variants": false,
+        "has_machine_materials": true,
+        "preferred_quality_type": "normal",
+        "has_machine_quality": true,

```

Comment:
As stated in https://github.com/Ultimaker/Cura/wiki/Definition-Files-Explained, the "has_machine_quality" should be set to true only if you have defined machine-specific quality files in the `resourses/quality/<machine_name>` folder.